src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -223,27 +236,19 @@ public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemp
     /// <param name="isPlayerSpecies">Whether this microbe is a member of the player's species</param>
     /// <param name="worldSettings">The world generation settings for this game</param>
     /// <param name="amountType">Specifies how changes during an in-game day are taken into account</param>
-    /// <param name="calculateRequiredResources">
-    ///   If true, then the required input compounds to run at the given energy balance are stored per energy producer
-    /// </param>
     /// <param name="cache">Auto-Evo Cache for speeding up the function</param>
-    public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
+    /// <param name="result">
+    ///   The resulting energy balance. If not null, the results are added to the passed value.
```

Comment:
```suggestion
    ///   The resulting energy balance.
```
Cannot be null anymore...